static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -19,24 +19,24 @@ import fastdom from 'fastdom';
 import raven from 'lib/raven';
 import userPrefs from 'common/modules/user-prefs';
 import images from 'common/modules/ui/images';
-import storage from 'lib/storage';
+import { localStorage } from 'lib/storage';
```

Comment:
Opera Mini is also a problem. But assuming localStorage is "nice to have" rather than necessary, perhaps it's worth just ensuring it exists before executing. Or not serving enhanced to Opera Mini?